src/storage/mongodb/MongoDBStorage.ts

```diff
@@ -258,6 +260,11 @@ export default class MongoDBStorage {
     // Logs
     await this.handleIndexesInCollection(Constants.DEFAULT_TENANT, 'logs', [
       { fields: { timestamp: 1 } },
+      { fields: { type: 1 } },
```

Comment:
Index 'type' is already handled in index { fields: { type: 1, timestamp: 1 } }
Same for 'action', 'level', source and host.

Reply:
Indexes are per field https://docs.mongodb.com/manual/core/index-single/ and compound indexes are only specific all fields together with an importance to the order: https://docs.mongodb.com/manual/core/index-compound

Reply:
Not at all!
![image](https://user-images.githubusercontent.com/25266945/106824834-fc070700-6683-11eb-9281-21006cbf5eaa.png)